### PR TITLE
Header

### PR DESCRIPTION
**Pull request by LarissaNamu**

### CODE REVIEW

File: club-website/src/components/ui/navigation-menu.tsx [hunk 1]
```diff
@@ -0,0 +1,128 @@
...
+    )}
+    {...props}
+  />
+))
+NavigationMenuList.displayName = NavigationMenuPrimitive.List.displayName
+
+const NavigationMenuItem = NavigationMenuPrimitive.Item
+
+const navigationMenuTriggerStyle = cva(
+  "group inline-flex h-10 w-max items-center justify-center rounded-md bg-background px-4 py-2 text-sm font-medium transition-colors hover:bg-accent hover:text-accent-foreground focus:bg-accent focus:text-accent-foreground focus:outline-none disabled:pointer-events-none disabled:opacity-50 data-[active]:bg-accent/50 data-[state=open]:bg-accent/50"
```
Comment (yclairew) [inline club-website/src/components/ui/navigation-menu.tsx:44]: to keep the frontend consistent with the figma, maybe replace "text-sm font-medium" with "text-xl font-bold"

File: club-website/src/components/header.tsx [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+  } from "@/components/ui/navigation-menu"
+
+export default function Header() {
+
+ 
+    return (
+        
+        <div className="flex justify-between p-6" id="logo_address">
+            <Link href="/about" legacyBehavior passHref>
+                <Image id="logo" src="/favicon180.png" alt="" width={50} height={50}></Image>
```
Comment (yclairew) [inline club-website/src/components/header.tsx:24]: not sure if i'm correct but should the logo on the header match the logo in the figma?

Comment (LarissaNamu) [reply club-website/src/components/header.tsx:24]: done!

File: club-website/src/components/header.tsx [hunk 3]
```diff
@@ -1 +1,62 @@
...
+
+export default function Header() {
+
+ 
+    return (
+        
+        <div className="flex justify-between p-6" id="logo_address">
+            <Link href="/about" legacyBehavior passHref>
+                <Image id="logo" src="/lightLogo.png" alt="" width={160} height={80}></Image> 
+            </Link>
```
Comment (peicasey) [inline club-website/src/components/header.tsx:25]: I don't think you need `legacyBehavior passHref` here, so you should get rid of it. Also I think the logo should link back to the root page so "/".

Comment (LarissaNamu) [reply club-website/src/components/header.tsx:25]: done ! :D

**Review by peicasey — APPROVED** [review]
lgtm

**Pull request merged**